logic/src/commonMain/kotlin/com/wire/kalium/logic/data/prekey/PreKeyRepository.kt

```diff
@@ -52,6 +53,11 @@ class PreKeyDataSource(
             proteusClientProvider.getOrCreate().newLastPreKey()
         }
 
+    override suspend fun getLocalFingerprint(): Either<ProteusFailure, ByteArray> =
+        wrapCryptoRequest {
+            proteusClientProvider.getOrCreate().getLocalFingerprint()
```

Comment:
please change it to getOrError
getOrCreate will create the local crypto files they are missing, we should make it clear in the doc when to use which